### ISSUE TITLE
Support InSpec attributes

### ISSUE THREAD

**Issue by chris-rock** [chef/kitchen-inspec#108]
InSpec introduced a new feature called attributes with InSpec 1.0 https://github.com/chef/inspec/issues/719. At this point, kitchen-inspec does not support this feature. My expectation is, that attributes work across the Chef tool chain.

a simple attribute https://github.com/chef/inspec/blob/master/examples/profile-attribute.yml file looks like 

```
user: bob
password: secret
```

To support attributes in kitchen.yml, we could do the following:

```
suites:
  - name: default
    verifier:
      inspec_tests:
        - https://github.com/dev-sec/tests-ssh-hardening
      attributes:
        user: bob
        password: secret
```

Not sure if we want to support external yml files?

**Comment by gsaslis** [comment]
👍  

also 👍  on the external yml files (in fact, I would even give this preference over the inline attributes declaration)

**Comment by chris-rock** [comment]
Related to #97

**Comment by chris-rock** [comment]
Fixed via #116
